cmake/FindSISL.cmake

```diff
@@ -1,13 +1,20 @@
 set(SISL_FOUND FALSE)
 
+
 if (NOT SISL_PREFIX)
     set(SISL_PREFIX ${CMAKE_INSTALL_PREFIX} CACHE FILEPATH "the installation prefix for the SISL library")
 endif()
 
+if(APPLE)
+  set(LIB_SUFFIX "dylib")
+else(APPLE)
+  set(LIB_SUFFIX "so")
+endif(APPLE)
+
 find_path(SISL_INCLUDE_DIRS "sisl.h"
     HINTS ${SISL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include)
 find_library(SISL_LIBRARIES
-    NAMES libsisl.so libsisl_opt.so libsisl.a libsisl_opt.a
+    NAMES libsisl.${LIB_SUFFIX} libsisl_opt.${LIB_SUFFIX} libsisl.a libsisl_opt.a
```

Comment:
Why not use CMAKE_SHARED_LIBRARY_SUFFIX which should work for all osses?
see http://www.cmake.org/Wiki/CMake_Useful_Variables